src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_NullCoalescingOperator.cs

```diff
@@ -47,14 +47,17 @@ public override BoundNode VisitNullCoalescingOperator(BoundNullCoalescingOperato
             // first we can make a small optimization:
             // If left is a constant then we already know whether it is null or not. If it is null then we 
```

Comment:
>If left is a constant [](start = 15, length = 21)

Should this comment be moved to line 60?

Reply:
I moved it to above 54, as that's where the optimization starts.

---
In reply to: [213727642](https://github.com/dotnet/roslyn/pull/29577#discussion_r213727642) [](ancestors = 213727642)